### PR TITLE
🚧 ⚰️ Authorize.net is dead: code

### PR DESCRIPTION
**Pull request by josemigallas**
[THREESCALE-6549: Remove all traces of 'authorize.net' from the code and the DB](https://issues.redhat.com/browse/THREESCALE-6549)

QUESTION: I needed to restart the server for Liquid to reload tags. What will happen in production?

TODOs
- [ ] Remove column from Oracle schema
- [ ] Check if any client is still using Authorize net
- [ ] Coordinate DB migration with ops, probably do in a separate PR

### CODE REVIEW

**Comment by akostadinov** [comment]
Well old pods are deleted and new pods are started with a new server.

File: db/migrate/20230224151935_remove_authorize_net_c_ctoken_from_account.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class RemoveAuthorizeNetCCtokenFromAccount < ActiveRecord::Migration[5.2]
+  def change
+    safety_assured { remove_column :accounts, :credit_card_authorize_net_payment_profile_token }
+  end
+end
```
Comment (jlledom) [inline db/migrate/20230224151935_remove_authorize_net_c_ctoken_from_account.rb:5]: According to this: https://github.com/ankane/strong_migrations#removing-a-column

The app should be restarted to be sure this migration doesn't cause problems, def. we'll need ops assistance to deploy this. But to be sure it doesn't cause any problem, another PR with only the `self.ignored_columns = %i[credit_card_authorize_net_payment_profile_token]` added to the `Account` model should be merged and deployed first.

File: lib/developer_portal/app/views/developer_portal/admin/account/authorize_net/credit_card_error.html.erb [hunk 2]
```diff
@@ -1,9 +0,0 @@
-Could not retrieve credit card info. Possible reasons:
-<ul>
-  <li>You don't
-  have <a href="http://www.authorize.net/solutions/merchantsolutions/merchantservices/cim/">Authorize.net
-  CIM enabled</a> . Make sure it is selected in your authorize.net
-  control panel.</li>
- <li>Merchant credentials are incorrect. Please double check your credentials are ok.</li>
- <li>Payment gateway is down.</li>
-</ul>
```
Comment (jlledom) [inline lib/developer_portal/app/views/developer_portal/admin/account/authorize_net/credit_card_error.html.erb:9]: Is this not required anymore? it seems to be not only exclusive to Authorize.net

Comment (josemigallas) [reply lib/developer_portal/app/views/developer_portal/admin/account/authorize_net/credit_card_error.html.erb:9]: Used only by authorized net:
```
git grep credit_card_error
lib/developer_portal/app/controllers/developer_portal/admin/account/authorize_net_controller.rb:      render action: 'credit_card_error'
```

File: lib/developer_portal/lib/liquid/tags/authorize_net_form.rb [hunk 3]
```diff
@@ -1,12 +0,0 @@
-module Liquid
```
Comment (mayorova) [inline lib/developer_portal/lib/liquid/tags/authorize_net_form.rb:1]: I guess we need to leave it here, the same way we did with the ogone form liquid.

**Comment by mayorova** [comment]
This needs quite some work to resolve all conflicts :smiling_face_with_tear: 

Maybe move it to Draft for now as it's not a priority?

**Comment by jlledom** [comment]
This one makes a year in a couple of days!

**Comment by josemigallas** [comment]
Closing in favor of smaller PRs, since migration needs it. Code removal now at https://github.com/3scale/porta/pull/3701

File: db/migrate/20230224151935_remove_authorize_net_c_ctoken_from_account.rb (hunk 1 above)
Comment (josemigallas) [reply db/migrate/20230224151935_remove_authorize_net_c_ctoken_from_account.rb:5]: https://github.com/3scale/porta/pull/3702

**Pull request closed without merging**